api/apps/services/provider_api_service.py

```
@@ -668,7 +668,23 @@ async def verify_api_key(provider_id_or_name: str, api_key: str | dict, base_url
+                    remote_models = await ModelMeta[provider_name](api_key, model_base_url).get_model_list()
+                    if remote_models:
+                        factory_llms = [
+                            {
+                                "model_type": mt,
+                                "llm_name": m["name"],
+                            }
+                            for m in remote_models
+                            for mt in m.get("model_types", [])
+                        ]
+            except Exception:
+                pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)